docs/proposals/35-observability2.0-victoria-logs.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Migrating from Vali to VictoriaLogs
+gep-number: 35
+creation-date: 2025-10-13
+status: implementable
+authors:
+- "@nickytd"
+- "@rrhubenov"
+reviewers:
+- "@plkokanov"
+- "@ScheererJ"
+- "@vpnachev"
+---
+
+# GEP-35: Migrating from Vali to VictoriaLogs
+
+## Table of Contents
+
+- [Terminology](#terminology)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Alternatives](#alternatives)
+
+## Terminology
+
+Since this document relies on work that has already been done in [GEP-34](./34-observability2.0-opentelemtry-operator-and-collectors.md), the same terminology applies here as well:
+- [OTLP](https://opentelemetry.io/docs/specs/otlp/): OpenTelemetry Protocol
+- [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/): A component that can receive, process, and export telemetry data (logs, metrics, traces).
+- [Vali](https://github.com/credativ/vali): A fork of Loki 2.2.1 that is used as the log database in the current Gardener observability stack.
+- [Loki](https://github.com/grafana/loki): A log database that is part of the Grafana ecosystem.
+
+## Summary
+
+This proposal introduces the deployment of [VictoriaLogs](https://github.com/VictoriaMetrics/VictoriaLogs) which will act as the replacement for [Vali](https://github.com/credativ/vali) in the Control Plane of all `Shoot` clusters that have the observability stack enabled, as well as in `Seed` clusters and `Garden` clusters.
+After the work that has been done on [GEP-34](https://github.com/gardener/gardener/pull/11861), we are in a favourable position to easily switch our storage layer for the clusters' log signals.
+
+This document fully relies on [GEP-34](./34-observability2.0-opentelemtry-operator-and-collectors.md) having already been implemented. 
+For this reason, it's expected that the reader is familiar with the concepts and terminology introduced in that GEP.
+
+A new operator is introduced that will be deployed to the `garden` namespace of `Seed` & `Garden` clusters.
```

Comment:
nit: My personal opinion is that in such design documents it is better to use natural language words instead of symbols with special meaning, when possible. 

```suggestion
A new operator is introduced that will be deployed to the `garden` namespace of `Seed` and `Garden` clusters.
```

PS. There few more places wher `&` can be replaced with `and`.